### ISSUE TITLE
[FR] Model choice not yet documented?

### ISSUE THREAD

**Issue by ozett** [jomjol/AI-on-the-edge-device#1874]
### The Feature

Updated from v12.0.1 to 13.0.8 
checked the documentation for update steps.
found under configuration a choice for models,
but only 2 of them [in the documentation](https://jomjol.github.io/AI-on-the-edge-device-docs/Choosing-the-Model/).

can i benefit from some models? What are the specs? When to choose which?
( found some hints [on the neural-types information](https://jomjol.github.io/AI-on-the-edge-device-docs/Neural-Network-Types/), it gets confusing..)

| which models|
|:---:|
![image](https://user-images.githubusercontent.com/5937811/213485942-dfe8de78-89d6-4357-8b6d-690926a31ca3.png)

digi-cont-xxx-seems versioning

**Comment by caco3** [comment]
Yes, the documentation is work in progress, there is quite some outdated stuff in it, sadly.

For selecting the model:
The 2 pages you found are what we have.

We usually suggest to just play with the models and see which one works best for you.
